std/uuid/v4.ts

```diff
@@ -11,11 +11,19 @@ export function validate(id: string): boolean {
   return UUID_RE.test(id);
 }
 
-export function generate(): string {
+export function generate(buf?: number[], offset?: number): string | number[] {
```

Comment:
@underfin please add a test case exercising new parameters

Reply:
Sorry for waste your time...